charts/partials/templates/_helpers.tpl

```diff
@@ -12,3 +12,12 @@ For example "11,22,55,44" will become "11","22","55","44"
 {{- end -}}
 {{- end -}}
 {{- end -}}
+
+{{/* vim: set filetype=mustache: */}}
```

Comment:
Also, probably best to keep editor-specific annotations out of source files.

Reply:
Removed [[ref]](https://github.com/linkerd/linkerd2/pull/12482#discussion_r1577913367)